docs/design/blocks/transformer.mdx

```diff
@@ -44,3 +44,120 @@ import pandas as pd
 def transform_df(df1, df2, df3, **kwargs) -> DataFrame:
     return pd.concat([df1, df2, df3])
 ```
+
+## Handling column renames
```

Comment:
this section is for data integration pipeline. could you update the section name to "Handling column renames in data integration pipeline"

Reply:
Absolutely